docs/src/tutorials/resolves.md

```diff
@@ -0,0 +1,627 @@
+Number of objective gradient evaluations             = 12
+Number of equality constraint evaluations            = 12
+Number of inequality constraint evaluations          = 0
+Number of equality constraint Jacobian evaluations   = 12
+Number of inequality constraint Jacobian evaluations = 0
+Number of Lagrangian Hessian evaluations             = 11
+Total seconds in IPOPT                               = 0.009
+
+EXIT: Optimal Solution Found.
+```
+As shown in the solver output, we've converged in 11 iterations!
+
+### Simulate the system
+Now that we've solved the optimal control problem, we'll extract the input for the next control step. This can be done via
+[`value`](@ref JuMP.value(::GeneralVariableRef)):
+```jldoctest quick
+julia> Tc_opt = value.(Tc)[2];
+```
+Next, the optimal input is used to simulate the system forward. Here, we choose to set up an ODE problem via `DifferentialEquations.jl`. Start by creating a function for the CSTR dynamics:
+```jldoctest quick
+julia> using DifferentialEquations
+
+julia> function cstrDynamics!(dx, x, p, t)
+        Ca, T, Tc = x
+        k = k₀ * exp(-E/T)
+        rate = k * Ca
+        dx[1] = F/V * (cf - Ca) - V * rate
+        dx[2] = (1/(rho * cp * V)) * (F * rho * cp * (Tf - T) + V * Hr * rate + Ua * (Tc - T))
+        end;
+```
+Then create a function that solves the ODE problem and returns the updated state:
+```jldoctest quick
+julia> function cstrSim(x0, tspan)
+        prob = ODEProblem(cstrDynamics!, x0, tspan)
+        sol = solve(prob, Tsit5(), reltol=1e-6, abstol=1e-8)
+        xsol = sol.u[end]
+        return xsol
+        end;
+```
+Now let's simulate the system forward!
+```jldoctest quick
+julia> tspan = (tk, tk + Δt); # Timespan to integrate over
+
+julia> xk = [init[1], init[2], Tc_opt];  # Current state + input
+
+julia> xsol = cstrSim(xk, tspan);    # Integrate the ODE problem
+
+julia> Ca_opt, T_opt = xsol[1:2];    # Obtain updated states
+```
+Note that the updated states `Ca_opt` and `T_opt` will become the initial conditions for the next optimal control solve.
+
+## Re-solves
+### Updating the model
+Now we need to update our model for the next solve. First, we update some parameters accordingly:
+```jldoctest quick
+julia> tk += Δt;     # Updated offset for the next control step
+```
+We'll then update the initial guesses in the model using the previous solution via [`warmstart_backend_start_values`](@ref).
+```jldoctest quick
+julia> warmstart_backend_start_values(model);
+```
+We'll also update the initial condition parameters using [`set_parameter_value`](@ref).
+```jldoctest quick
+julia> set_parameter_value(Ca0, Ca_opt);
+
+julia> set_parameter_value(T0, T_opt);
+```
+Lastly, we'll need to update the setpoint parameter function, which can also be done via [`set_parameter_value`](@ref). In this case, we'll need to create a new function with an updated offset value.
+```jldoctest quick
+julia> newTsp = (t) -> setpoint(t, tk);
+
+julia> set_parameter_value(Tsp, newTsp);
```

Comment:
This is the whole focus of the re-solve interface some more discussion is needed. Better yet, I would detail all these functions at the beginning of the tutorial to better define what the re-solve interface is.